packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_FR.json

```diff
@@ -835,5 +835,9 @@
   "domain_dashboard_recommended_hosting_title_Perso": "Hébergement de votre blog ou votre site web",
   "domain_dashboard_recommended_hosting_label": "-{{ reduction }} sur votre Hébergement {{ type }} pour le Black Friday.",
   "domain_dashboard_recommended_hosting_discover": "Découvrir les solutions",
-  "domain_tab_emails": "E-mails et mailing-lists"
+  "domain_tab_emails": "E-mails et mailing-lists",
+  "domain_dashboard_general_information_domain_terminated": "Résiliation demandée",
+  "domain_dashboard_general_information_domain_automatic_renewal": "Renouvellement automatique",
+  "domain_dashboard_general_information_alldom_termination_info": "Votre pack expirera le {{ expirationDate }}. Les domaines non résiliés inclus dans celui-ci seront <strong>renouvelés gratuitement jusqu’à l’expiration définitive de votre pack.</strong> Au-delà de cette date, vos domaines non résiliés seront facturés individuellement.",
```

Comment:
**nitpick**:
The whole sentence `Les domaines non résiliés […] votre pack` could be set in the `<strong>`. Also, this could be splitted to extract the HTML tag from the translation content.

Reply:
@JeremyDec according to the mockup only the part of the sentence is bold because only the info regarding the payment needs to be emphasised.